Representable/representable
# public comment links updated, map visualization style updated, tags bugfix

**changes**
- added public comment links for AK, AR, GA, HI, LA, NM, NV, NY, OR, PA, SC, UT, VT
- fixed bug where tags suggestions would appear behind the "back" and "next" buttons on entry page
- updated map pages visualization to have a more colorful style

**to test**
- python manage.py runserver
- draw community in one of the public comment states above ^
- check tags bug isn't happening
- check public comment card appears on submission page
- check map pages (state + drive + org) have updated style and no errors
